superset/sql_lab.py

```diff
@@ -190,6 +190,10 @@ def execute_sql_statement(
     db_engine_spec = database.db_engine_spec
     parsed_query = ParsedQuery(sql_statement)
     sql = parsed_query.stripped()
+    # This is a test to see if the query is being
+    # limited by either the dropdown or the sql.
+    # We are testing to see if more rows exist than the limit.
+    increased_query = None if query.limit is None else query.limit + 1
```

Comment:
Can you rename this to `increased_limit`? The current name suggests that this is a query, but it's actually a limit.